src/components/EditorSelectBox.js

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+
+import styled from "styled-components";
+
+const Select = styled.select`
+  appearance: none;
+  width: 140px;
+  height: 40px;
+  padding: 5px 30px 5px 10px;
+  border: none;
+  border-radius: 4px;
+  outline: 0 none;
+  background: black url("images/grey_arrow.png") calc(100% - 5px) center no-repeat;
+  color: #8E9093;
+  font-size: 20px;
+  font-family: Inter;
+  font-weight: bold;
+`;
+
+const SelectOption = styled.option`
+  padding: 3px 0;
+  background: black;
+`;
+
+export default function EditorSelectBox({ type, options }) {
+  const [selected, setSelected] = useState("default");
+
+  const handleChangeSelect = (event) => {
+    setSelected(event.target.value);
+  };
+
+  return (
+    <Select name="sort" className="select" onChange={handleChangeSelect} value={selected}>
+      <SelectOption value="default" disabled>{type}</SelectOption>
+      {options.map((theme) => {
+        return <SelectOption value={theme}>{theme}</SelectOption>;
```

Comment:
저는 다른 컴포넌트 생성 시 해당 로직을 한 줄로 작성했습니다.

이 부분은 상의 후 스타일을 통일하면 좋을 것 같아요!😊

Reply:
좋습니다! 저는 코드 한 줄을 짧게 가져가는 편이어서 보통 중괄호 앞에서 띄워주는 식으로 작업합니다.
다른 분들은 어떠신가요?

Reply:
이렇게 작성하신다고 합니다! 생각해보니 저도 전에 과제들 진행할 때 이렇게 작성했던 기억이 있구요!

```suggestion
      {options.map((theme) => (
        <SelectOption value={theme}>
          {theme}
        </SelectOption>
      )}
```

Reply:
화살표 뒤에서 띄우되 중괄호+리턴문이 아닌 괄호 리턴으로 처리하신다는 거죠? 그럼 앞으로 이렇게 가도록 할까요!

Reply:
네네 단오님이 동의하시면 그렇게 통일할까 합니다!👍

Reply:
좋습니다!